### ISSUE TITLE
Docker-Compose file?

### ISSUE THREAD

**Issue by lonnietc** [RasaHQ/rasa-demo#502]
Hello,

Is there a docker-compose file to bring up the Sara Chatbot from scratch?

I would like to run it locally in a docker container to see it working.
Thanks

**Comment by akelad** [comment]
There isn't one part of the repo at the moment no. Could you do it using the run instructions in the readme?

**Comment by lonnietc** [comment]
Hello,

Thanks for responding to my inquiry.

I actually did come across a docker-compose setup for a simple Rasa Chatbot (https://github.com/JiteshGaikwad/Rasa_Deployment) which boots up beautifully.

With that, I tried to move over the Sara files into and build it up from scratch, but although it booted with docker-compose, Sara did not seem to function in the Web UI.

This means that I just do not know enough yet to be able to swap out chatbot models.

Maybe the Rasa team could use the "Rasa_Deployment" stuff to get Sara working from a docker standpoint if it is helpful.

I really like what I have read and am learning about Rasa and have a GREAT project that I want to use Rasa for but do not have the core skills yet and may need to try and locate someone that can help for this small Proof-Of-Concept project.

Thanks again and stay safe
